### PR TITLE
Added greetings & image <3

### PR DESCRIPTION
**Pull request by lometari**
Trying pull-request

### CODE REVIEW

**Comment by artem-trubin** [comment]
Hi and welcome! Can you please post the image with Markdown syntax, not with an HTML tag?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Kottans-frontend
 # Help with your first commit -> [discord](https://discord.gg/k5zS4XmH)
 ___
+## **Hi there!**
+Despite I've got only paws I'm looking forward to learn and try something new and interesting!
+
+<img src="https://github.com/lometari/mock-repo/raw/firstStep/gif/belovedCat.jpg" width="70%" margin="auto">
```
Comment (suchov) [inline README.md:7]: no HTML tegs in markdown pls 
you can read more about hot to do it here -> https://www.markdownguide.org/basic-syntax/

**Review by suchov — CHANGES_REQUESTED** [review]

**Comment by lometari** [comment]
Hi! Should I need to change something else?

**Comment by suchov** [comment]
Hi, nope, looks like everything is amazing.

**Pull request merged**